ui/tests/acceptance/setup.js

```diff
@@ -0,0 +1,11 @@
+import path from 'path'
+const PHOENIX_PATH = process.env.PHOENIX_PATH
+
+console.log(PHOENIX_PATH, "/tests/acceptance/setup.js")
+
+import {init} from "/home/dipakacharya/www/phoenix/tests/acceptance/setup"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not import modules using an absolute path](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=5709816)